README.md

```diff
@@ -74,79 +76,114 @@ docker exec -t sphinx bash /root/index.all.sh
-1. Move up and over into the `vanilla-docker` directory.
-1. Run `sudo ./mac-setup.sh` which will:
-    - Add a self signed certificate `*.vanilla.localhost` to your keychain.
-    - Safely update your `/etc/hosts`.
-    - Add `192.0.2.1` as a loopback IP address.
-    - Create a docker volume named "datastorage" which will contain the database data.
-1. Run `docker-compose up --build` (It will take a while the first time). You'll know it worked if you see something like
-    ```
-    Successfully built…
-    Successfully tagged…
-    Creating database ... done
-    Creating php-fpm  ... done
-    Creating httpd    ... done
-    Creating nginx    ... done
-    Attaching to database, php-fpm, httpd, nginx
-    …
-    php-fpm     | done.
-    ```
-    and voila -> [dev.vanilla.localhost](https://dev.vanilla.localhost/) shows the Vanilla installer.
+1. Run `vanilla-start` (It will take a while the first time). You'll know it worked if you see something like
+   ```
+   Successfully built…
+   Successfully tagged…
+   Creating database ... done
+   Creating php-fpm  ... done
+   Creating httpd    ... done
+   Creating nginx    ... done
+   Attaching to database, php-fpm, httpd, nginx
+   …
+   php-fpm     | done.
+   ```
+   and voila -> [dev.vanilla.localhost](https://dev.vanilla.localhost/) shows the Vanilla installer.
 1. Run the installer!
-    
-    - It is recommended to use `vanilla_dev` as the database name since some services are configured to use that database.
-    - It is recommended to use `database` for the host name.
-    - It is recommended to use `root` as the username.
+   - It is recommended to use `vanilla_dev` as the database name since some services are configured to use that database.
+   - It is recommended to use `database` for the host name.
+   - It is recommended to use `root` as the username.
+
+### Stopping Docker
+
+To properly stop the containers run `vanilla-stop`
+
+## Updating an existing vanilla-docker installation
+
+- Pull the latest version of `vanilla-docker`
+- Restart docker entirely.
 
-To properly stop the containers you need to run `docker-compose down`.
+  ![image](https://user-images.githubusercontent.com/1770056/82736370-4a8f9380-9cf7-11ea-8f18-26f965e9abe1.png)
 
-Do not forget to run `docker-compose up --build` to start up the services every time you restart your computer.
+- Navigate to `vanilla-docker`
+- Run `bin/mac-setup` (even if you already ran it before).
+- Run `vanilla-start` from anywhere.
+
+### Updating to a RAW data format
+
+You may see a message "It looks like your docker volume is not in the RAW format." when trying to proceed with your upgrade.
+
+In order to changer to the RAW data format for docker, you need to do the following:
+
+- Backup any databases you have.
+- Open the docker troubleshooting page. (Open docker settings, click the "Bug" icon next to settings.)
+- Click on `Clean / Purge data` ~`Reset disk image`.~ (It was renamed recently)
```

Comment:
I tried this on Tuan's computer, but it didn't work. I don't know what the issue is.